### DISCUSSION TITLE
question: renovate making PRs for dependency marked as `chore` instead of `fix`

### DISCUSSION THREAD

**Discussion by RealShadowNova** [renovatebot/renovate#17449] [Request Help] [answered]
I've been using Renovate for awhile and recently realized that when it bumps both `devDependencies` and `dependencies` it is not marking the PR as `fix` and rather as `chore`. Anyone know why that would be?

## Relevant Sources

- [josh-development/website#28](https://github.com/josh-development/website/pull/28)
- [Base Configuration](https://github.com/josh-development/.github/blob/main/josh-renovate.json)
- [Local Configuration](https://github.com/josh-development/website/blob/main/.github/renovate.json)

**Comment by Gabriel-Ladzaretti** [comment]
You are probably using the following preset as part of the base preset.
https://docs.renovatebot.com/presets-default/#semanticprefixfixdepschoreothers

packageRules are evaluated one by one, so the semantic `fix` rule is the one set in this case.

**Comment by rarkins** [comment, marked as answer]
Renovate doesn't know/understand that `fix` is more important than `chore`. So when there are multiple update types in a group then Renovate will use the prefix of whichever update is first in the list. If you plan to group together updates with different types/prefixes, you should add an extra rule to force `fix` on the group (yes, even if they're all dev).
